charts/jobs/templates/job.yaml

```diff
@@ -14,6 +14,11 @@ spec:
       annotations:
         checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") $ | sha256sum | trunc 63 }}
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") $ | sha256sum | trunc 63 }}
+        {{- if $spec.annotations }}
```

Comment:
remove the condition wrapping as iterating over an empty array won't print anything